src/github/utils/config.ts

```diff
@@ -29,13 +29,13 @@ export async function getConfig(context: GitHubContext): Promise<BotConfig | nul
   try {
     config = Value.Decode(configSchema, Value.Default(configSchema, _repoConfig));
   } catch (error) {
-    console.error("Error decoding config", error);
-    return null;
+    console.error("Error decoding config, will use default.", error);
+    return defaultConfiguration;
   }
 
   checkPluginChains(config);
 
-  return generateConfiguration(config as BotConfig);
+  return generateConfiguration(Value.Default(configSchema, _repoConfig) as BotConfig);
```

Comment:
why not use `config`, it already has defaults?

Reply:
The problem with using config is that on Decode, the field for plugin is changed from one string to an object with repo, owner, etc. This susequently does not match the original schema which is designed to receive a string.

Reply:
What do you mean by original schema? This function should return plugin with object and not string

Reply:
`generateConfiguration` expects string for the `plugin` field, e.g. `ubiquibot/conversation-rewards@main`. Once `Decode` is called, it turns in to an object of the shape
```TypeScript
{
  owner: string;
  repo: string;
  workflowId: string;
  ref?: string;
};
```
So passing `config` again through the `generateConfiguration` fails because of this. But in the end returning just `config` should be enough.